module/os/linux/zfs/spa_misc_os.c

```diff
@@ -42,6 +42,19 @@
 #include <sys/kstat.h>
 #include "zfs_prop.h"
 
+
+int
+param_set_deadman_failmode(const char *val, zfs_kernel_param_t *kp)
```

Comment:
The `module_param_call` function should be switched to use the `ZFS_MODULE_PARAM_CALL` wrapper.

```diff
diff --git a/module/zfs/spa_misc.c b/module/zfs/spa_misc.c
index 3956fc4..6af6c95 100644
--- a/module/zfs/spa_misc.c
+++ b/module/zfs/spa_misc.c
@@ -2847,11 +2847,9 @@ ZFS_MODULE_PARAM(zfs, zfs_, ddt_data_is_special, INT, ZMO
 ZFS_MODULE_PARAM(zfs, zfs_, user_indirect_is_special, INT, ZMOD_RW,
        "Place user data indirect blocks into the special class");
 
-#ifdef _KERNEL
-module_param_call(zfs_deadman_failmode, param_set_deadman_failmode,
-       param_get_charp, &zfs_deadman_failmode, 0644);
-MODULE_PARM_DESC(zfs_deadman_failmode, "Failmode for deadman timer");
-#endif
+ZFS_MODULE_PARAM_CALL(zfs_deadman, zfs_deadman_, failmode,
+       param_set_deadman_failmode, param_get_charp, ZMOD_RW,
+       "Failmode for deadman timer");
 
 /* BEGIN CSTYLED */
 ZFS_MODULE_PARAM_CALL(zfs_deadman, zfs_deadman_, synctime_ms,
```